sqlmesh/dbt/loader.py

```diff
@@ -148,7 +148,7 @@ def __init__(
             self._macros_max_mtime = macros_max_mtime
             self._yaml_max_mtimes = yaml_max_mtimes
 
-            cache_path = loader._context.path / c.CACHE / project.context.target.name
+            cache_path = loader._context.path / c.CACHE / project.context.target.name  # type: ignore
```

Comment:
Should we raise if the target is `None` instead?

Reply:
It's a mypy thing, so replaced with a cast.